app/models/creditsandrefunds/CreditAndRefundViewModel.scala

```diff
@@ -16,18 +16,35 @@
 
 package models.creditsandrefunds
 
-import models.creditDetailModel.{BalancingChargeCreditType, CreditType, CutOverCreditType, MfaCreditType}
-import models.financialDetails.{DocumentDetailWithDueDate, FinancialDetail}
+import models.financialDetails._
+import play.api.i18n.Messages
 
+import java.time.LocalDate
 
-case class CreditAndRefundViewModel(creditCharges: List[(DocumentDetailWithDueDate, FinancialDetail)]) {
-
+sealed trait CreditRow {
+  val amount: BigDecimal
+  val creditType: CreditType
+}
+case class CreditViewRow(amount: BigDecimal, creditType: CreditType, taxYear: String) extends CreditRow
```

Comment:
ideally if we can use TaxYear type instead of String type, but should be ok this time as there already lots of changes in scope of this story / feel free to raise Tech Debt

Reply:
Shall I raise the ticket for updating TaxYear type across the repo or just for these files?

Reply:
I think it would be easy to approach this case by case, but yes, you can move this change into tech debt if you like, scope TBC